src/commands/lambda/flare.ts

```diff
@@ -30,6 +30,18 @@ const LOGS_DIRECTORY = 'logs'
 const FUNCTION_CONFIG_FILE_NAME = 'function_config.json'
 const ZIP_FILE_NAME = 'lambda-flare-output.zip'
 const LOG_STREAM_COUNT = 3
+const FULL_OBFUSCATION = '****************'
+const MIDDLE_OBFUSCATION = '**********'
```

Comment:
This constants since one is the subset of the other.
And this can be easily calculated within the method that masks data.

Why don't we go for a different approach?

Reply:
Resolved as we talked about it in Slack